docs/3.0rc/community/install-contribute.mdx

```diff
@@ -0,0 +1,282 @@
+
+```
+prefect dev start
+```
+
+Start a Prefect API that reloads on code changes:
+
+```
+prefect dev api
+```
+
+
+```
+
+### UI development
+
+Developing the Prefect UI requires installation of [npm](https://github.com/npm/cli).
+
+Start a development UI that reloads on code changes:
+
+```
+prefect dev ui
+```
+
+Build the static UI (the UI served by `prefect server start`):
+
+```
+prefect dev build-ui
+```
+
+### Docs Development
+
+Prefect uses [mkdocs](https://www.mkdocs.org/) for the docs website and the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) 
+theme. While we use `mkdocs-material-insiders` for production, builds can still happen without the extra plugins. Deploy previews 
+are available on pull requests, so you can browse the final look of your changes before merging.
+
+To build the docs:
+```
+mkdocs build
+```
+
+To serve the docs locally at [http://127.0.0.1:8000/](http://127.0.0.1:8000/):
+```
+mkdocs serve
+```
+
+For additional mkdocs help and options:
+```
+mkdocs --help
+```
+
+We use the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. To add additional JavaScript or CSS to the docs, 
+please see the theme documentation [here](https://squidfunk.github.io/mkdocs-material/customization/).
+
+Internal developers can install the production theme by running:
+
+```
+pip install -e git+https://github.com/PrefectHQ/mkdocs-material-insiders.git#egg=mkdocs-material
+mkdocs build # or mkdocs build --config-file mkdocs.insiders.yml if needed
+
+```
+
+
+### Kubernetes development
+
+Generate a manifest to deploy a development API to a local kubernetes cluster:
+
+```
+prefect dev kubernetes-manifest
+
+```
+
+
+To access the Prefect UI running in a Kubernetes cluster, use the `kubectl port-forward` command to forward a port on your local 
+machine to an open port within the cluster. For example:
+
+```
+kubectl port-forward deployment/prefect-dev 4200:4200
+
+```
+
+
+This forwards port 4200 on the default internal loop IP for localhost to the Prefect server deployment.
+
+To tell the local `prefect` command how to communicate with the Prefect API running in Kubernetes, set the `PREFECT_API_URL` 
+environment variable:
+
+```
+export PREFECT_API_URL=http://localhost:4200/api
+
+```
+
+
+Since you previously configured port forwarding for the localhost port to the Kubernetes environment, you can interact 
+with the Prefect API running in Kubernetes when using local Prefect CLI commands.
+
+### Add database migrations
+
+To make changes to a table, first update the SQLAlchemy model in `src/prefect/server/database/orm_models.py`. For example, 
+to add a new column to the `flow_run` table, add a new column to the `FlowRun` model:
+
+```
+# src/prefect/server/database/orm_models.py
+
+@declarative_mixin
+class ORMFlowRun(ORMRun):
+    """SQLAlchemy model of a flow run."""
+    ...
+    new_column = Column(String, nullable=True) # <-- add this line
```

Comment:
This line and next are showing up as problematic in GitHub. Is it looking okay in local deb server?